src/components/article-body/body-components/list.tsx

```diff
@@ -0,0 +1,26 @@
+import { ComponentFactory } from '../component-factory';
+import React from 'react';
+import { ArticleNode } from '../../../interfaces/article-body-node';
+
+export const List = ({
+    children,
+    ordered,
+    ...rest
+}: {
+    children: any;
+    ordered: boolean;
+}) => {
+    return !ordered ? (
+        <ul>
+            {children.map((child: ArticleNode, index: number) => (
+                <ComponentFactory {...rest} key={index} nodeData={child} />
```

Comment:
Don't use the index as the key as it could cause issues in React https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067